tests/xbot/common/properties/PermanentStorageProxyTest.java

```diff
@@ -42,6 +42,20 @@ public void testSaveAndLoad() {
         assertEquals("What time is it?", p.getString("phrase"));        
     }
     
+    @Test
+    public void testClear() {
```

Comment:
This test makes me a little weary since it could pass even if clear didn't work but the write to file/load from disk part wasn't working.